### DISCUSSION TITLE
Simple way to add non-standard Header names

### DISCUSSION THREAD

**Discussion by Harald2019F** [spring-projects/spring-integration#3731] [Q&A]
Hello,
do you know a simple way how to add a non-standard Header name when using spring integration (inbound gateway). It looks like default class DefaultHttpHeaderMapper might be a key element but how can you gain an additional header name to the header. When I send a header with a name that is not included in the Spring Integration standard list of header names it is ignored. I want to use an additional one in a simple  way.

Thanks a lot 
Harald

**Comment by artembilan** [comment]
I'm going to answer to the same your question on StackOverflow: https://stackoverflow.com/questions/71193184/passing-non-standard-header-names-in-spring-integration-inbound-gateway
